client/src/components/App/Content/LandingPage/adminLandingPage/UnassignedCard/UnassignedCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card, Tooltip, Typography } from '@material-ui/core';
+import NavigateBeforeIcon from '@material-ui/icons/NavigateBefore';
+
+import useStyles from './UnassignedCardStyles';
+
+const unnasignedInvestigationsText = 'חקירות לא משויכות/ משויכות לחוקרים לא פעילים';
+
+const UnassignedCard: React.FC = (): JSX.Element => {
+    const classes = useStyles();
+
+    let history = useHistory();
+
+    return (
+        <Tooltip title={unnasignedInvestigationsText}>
+            <Card className={classes.unassignedCard} onClick={() => {}}>
+                <div className={classes.investigationAmount}>
+                    <Typography className={classes.investigationNumberText}><b>13</b></Typography>
```

Comment:
Should the data be hard-coded for now?

Reply:
yes in the future there will be integration with DB